src/main/java/org/terasology/magicalstats/system/MagicalStatsSystem.java

```diff
@@ -111,4 +111,6 @@ public void onArcaneRESChanged(OnArcaneResistanceChangedEvent event, EntityRef p
         }*/
     }
 
+
+
```

Comment:
Extra lines with no other changes here is kinda weird, not that it really hurts. On a related note if you see the funny little read symbol at the bottom of files like this that's GitHub encouraging you to put empty lines at the bottom of any sort of text file for some sort of good practice reason. So feel free to aim for that / fix similar places if you notice.